### PR TITLE
docs: add link to contact page

### PR DESCRIPTION
**Pull request by BogdanDanilescu**
image

### CODE REVIEW

File: apps/docs/src/app/contact/page.tsx [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
       <Heading>Contact</Heading>
       <Paragraph>
-        You can contact the design system team via email. Email coming soon.
+        You can contact the design system team using{' '}
+        <a
+          target="_blank"
+          className="underline text-blue-700 hover:decoration-md"
+          href={config.signUpFormUrl}
+        >
+          this form.
```
Comment (ianlnf) [inline apps/docs/src/app/contact/page.tsx:15]: period can be moved outside of link

File: apps/docs/src/app/contact/page.tsx [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+import { config } from '@/lib/config';
 
 export default function ContactPage() {
   return (
     <div>
       <Heading>Contact</Heading>
       <Paragraph>
-        You can contact the design system team via email. Email coming soon.
+        You can contact the design system team using{' '}
+        <a
```
Comment (ianlnf) [inline apps/docs/src/app/contact/page.tsx:10]: could add a TODO to replace with the `Link` component when it supports target (for styling consistency)

Comment (hamza14khan) [inline apps/docs/src/app/contact/page.tsx:10]: nitpick: You can use `<Link>` component from `govie-react/ds`.

Comment (BogdanDanilescu) [reply apps/docs/src/app/contact/page.tsx:10]: Good idea. Though if we use the `Link` component from `govie-react/ds` we won't be able to open the form in a new tab as it doesn't have a `target` prop yet

Comment (BogdanDanilescu) [reply apps/docs/src/app/contact/page.tsx:10]: maybe we should add a TODO as Ian was suggesting

**Review by ianlnf — APPROVED** [review]

**Pull request merged**